go/protocol/keybase1/extras_test.go

```diff
@@ -33,6 +33,30 @@ func TestTime(t *testing.T) {
 	}
 }
 
+func TestTimeConversions(t *testing.T) {
+	times := []time.Time{
+		time.Time{},
+		time.Now(),
```

Comment:
also test for something like
```
		time.Now().Add(time.Duration(12096123 * time.Second)),
		time.Now().Add(time.Duration(-12096321 * time.Second)),
```
both keybase1.UnixTime and keybase1.Time should be precise at least to a second